apm-lambda-extension/logsapi/subscribe.go

```diff
@@ -29,9 +29,9 @@ import (
 	"github.com/pkg/errors"
```

Comment:
I exposed `listenerHost`, `logsAPIServer` and `logsAPIListener` for testing purposes. The linter recommended against using the package name in variable names, hence the renaming of `logsAPIServer` and `LogAPIListener`